modules/openapi-generator/src/main/resources/typescript/http/http.mustache

```diff
@@ -243,9 +243,12 @@ export class ResponseContext {
             return result;
         }
 
-        const parameters = this.headers[headerName].split(";");
+        const parameters = this.headers[headerName]!.split(";");
```

Comment:
this is guarded further up, but some of the older TS versions we have can't determine this, yet.